### DISCUSSION TITLE
Is there any way to reload the extensions without restarting the process?

### DISCUSSION THREAD

**Discussion by mattiaz9** [directus/directus#8851] [Feature Requests]
I remember few months ago, when I was working on my extensions in watch mode, it used to update them on page refresh, so I could work without restarting the process every time.

Now I have to stop the process and restart it every time a make changes to the extensions and this is slowing me down.

Is there some settings that enables to update changes without restarting the process?

**Comment by joselcvarela** [comment]
Hello @mattiaz9 
You can use the `directus-extension build --watch` https://github.com/directus/directus/discussions/8190#discussioncomment-1390753
We are working on a proper development flow for extensions but for now you can try to use that option.

**Reply by mattiaz9** [reply]
Yes, I do use the `--watch` option, but my problem is that I have to restart the process for directus to apply the changes.

Nice to hear you are working on a better flow development. I hope you can improve this aspect :)
